fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/rollcall/CloseRollCall.java

```diff
@@ -0,0 +1,95 @@
+package com.github.dedis.student20_pop.model.network.method.message.data.rollcall;
+
+import com.github.dedis.student20_pop.model.network.method.message.data.Action;
+import com.github.dedis.student20_pop.model.network.method.message.data.Data;
+import com.github.dedis.student20_pop.model.network.method.message.data.Objects;
+import com.github.dedis.student20_pop.utility.protocol.DataHandler;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Data sent to close a Roll-Call */
+public class CloseRollCall extends Data {
+
+  private final String id;
```

Comment:
this will need updating based on our earlier discussion today :)

Reply:
To be taken up in a separate PR